### DISCUSSION TITLE
Reporting license violation

### DISCUSSION THREAD

**Discussion by irepoetry** [plausible/analytics#3178] [Anything Else]
### Past Issues Searched

- [X] I have searched open and closed issues to make sure that the bug has not yet been reported

### Issue is a Bug Report

- [X] This is a bug report and not a feature request, nor asking for self-hosted support

### Using official Plausible Cloud hosting or self-hosting?

Plausible Cloud from plausible.io

### Describe the bug

[bayanplus.co](https://bayanplus.co) uses Plausible's source but so far has refused to share back their changes and improvements that enable RTL support and Arabic interface. They've claimed that they've built their own code from scratch, which doesn't seem to be true.

According to the AGPL v3 license, which Plausible is released under, bayanplus.co should:

1. Share modifications under the same license.
2. Not remove the original author's copyright.

However, I honestly don't care about anything other than giving credit to those who deserve it. My intent is not to cause trouble for the people at Bayanplus. Instead, the hard work done by open-source software developers should be respected.

### Expected behavior

1. Share modifications under the same license.
2. Not remove the original author's copyright.

**Comment by metmarkosaric** [comment]
thanks for reporting @irepoetry! will check it out

(moving this from issues as we keep issues to the actual bugs with the product only)

**Comment by hadysata** [comment]
Hello @irepoetry,

I am writing to address your claim about Bayan Plus violating the license of Plausible.

As the CTO of a tech company called Hevea, I am in a position to address this issue as we have been using Bayan Plus from the inception of their platform. I have personally been privy to their development process, their progress, and their efforts to enhance the platform right from the start. I can vouch for the fact that they have built their platform from scratch and when they have utilized any open-source project, they have not failed to give due credit.

What's important to understand here is that Bayan Plus and Plausible are fundamentally different in their tech stacks and underlying technologies. The claim that Bayan Plus has used the source code from Plausible is not only unfounded but also undermines the genuine hard work and unique innovation Bayan Plus has put into their platform. 

The AGPL v3 license promotes the free use, distribution, and modification of the software as long as the original work is attributed. Bayan Plus, although it may appear similar to Plausible, is built with different technologies. It is crucial not to mistake similarities in functionality as an indication of code plagiarism.

I understand and appreciate your intention to respect and protect the work of open-source software developers. However, it is equally important to ensure that we do not rush to false judgments or unfounded accusations that might tarnish the reputation of hardworking teams like Bayan Plus.
